test/integration/fixtures/test_dynamic.html

```diff
@@ -0,0 +1,15 @@
+<html>
+
+<body>
+  <p id="print-ready" ready-to-print="false"></p>
```

Comment:
No specific reason. The first version was in fact just following that the attribute is set, later I thought that perhaps a value could be useful. I'm happy with the idiomatic way, just following when the attribute appears. It would be more user-friendly, because there is no risk of typos in the value.

I can refactor this to just follow the attribute, would that be good approach?

Reply:
🤔  I can't think of a use-case right now where the value would be truly needed, and as far as I can tell it's currently also not optional, so I would say yes, let's go without it.

Reply:
another thought: Would it make sense to also give a really short sample of JavaScript in the documentation that triggers the attribute? I'm assuming you're using it with `window.onload` or so?

maybe sth. like

```
# the corresponding JS code in the markup
<script>
  window.onload = function() {
    // dynamic content loading...
    document.getElementById('print-ready').setAttribute('ready-to-print', 'true');
  };
</script>
```

Reply:
Yeah, I think that a documentation example would be good addition to the docs here. That's basically the gist of it: one could also follow the fonts being loaded etc.